gp-includes/warnings.php

```diff
@@ -401,8 +620,123 @@ function ( $key ) {
 			}
 		);
 
-		foreach ( $warnings as $warning ) {
-			$translation_warnings->add( str_replace( 'warning_', '', $warning ), array( $this, $warning ) );
+		$warnings = array_fill_keys( $warnings, $this );
+
+		$warnings = apply_filters( 'gp_add_all_warnings', $warnings );
+
+		foreach ( $warnings as $warning => $class ) {
+			$translation_warnings->add( str_replace( 'warning_', '', $warning ), array( $class, $warning ) );
 		}
 	}
+
+	/**
+	 * Returns the values from the href and the src
+	 *
+	 * @since 3.0.0
+	 * @access public
```

Comment:
```suggestion
	 * @access private
```

Can you review the `@access` tags again? There are a few others which don't match the actually visibility.

To be honest, I'm fine if you just keep them out.

Reply:
I have updated the visibility to two functions, to maintain the consistency with the other functions, that have the "@access" parameter